### PR TITLE
P1 list

### PR DESCRIPTION
**Pull request by ritu9898**

### CODE REVIEW

File: components/style/list.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  width: 20rem;
+  background-color: var(--white);
+  display: flex;
+  justify-content: flex-start;
+  align-items: baseline;
+}
+
+.list .details {
+  padding: 0px 15px;
+  word-spacing: 2px;
```
Comment (samyakshah3008) [inline components/style/list.css:12]: Good use of shorthand property

File: components/style/list.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.list {
+  box-shadow: 0px 0px 1px var(--grey);
+  width: 20rem;
```
Comment (samyakshah3008) [inline components/style/list.css:3]: You can give min or max width for responsiveness

**Review by samyakshah3008 — APPROVED** [review]
Good use of semantic tags, overall code is good 
Suggestions added✅ 
Also you can provide PR preview description for better code review experience 

Rest is all good for approval ✅

File: components/style/list.css (hunk 2 above)
Comment (ritu9898) [reply components/style/list.css:3]: Ya, will do that

**Pull request merged**